### ISSUE TITLE
rustfmt failing to build on stable 1.26 - extprim failing

### ISSUE THREAD

**Issue by hammerandtongs** [rust-lang-nursery/rustfmt#2701]
extprim build is keeping rustfmt from building on stable 1.26 over the new u128 support in stable.

https://github.com/kennytm/extprim/issues/15

**Comment by nrc** [comment]
extprim has been fixed in v1.6, but I don't see where we depend on extprim - I don't see an entry for it in our Cargo.lock

**Comment by nrc** [comment]
> Will rustup keep this uptodate now?

Yes, as long as you `rustup update`, Rustfmt will be kept up to date.

**Comment by hammerandtongs** [comment]
Should cargo stop trying to install rustfmt? Is that a bug on cargo?

**Comment by nrc** [comment]
I think you need to explicitly uninstall it with Cargo
